### ISSUE TITLE
Crash report 5.6: ArrayIndexOutOfBoundsException in SitePickerAdapter

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#4509]
```
Fatal Exception: java.lang.ArrayIndexOutOfBoundsException: length=12; index=-1
       at java.util.ArrayList.get(ArrayList.java:310)
       at org.wordpress.android.ui.main.SitePickerAdapter.getItem(SitePickerAdapter.java:127)
       at org.wordpress.android.ui.main.SitePickerAdapter.access$300(SitePickerAdapter.java:32)
       at org.wordpress.android.ui.main.SitePickerAdapter$SiteViewHolder$1.onClick(SitePickerAdapter.java:89)
       at android.view.View.performClick(View.java:5197)
       at android.view.View$PerformClick.run(View.java:20926)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:145)
       at android.app.ActivityThread.main(ActivityThread.java:5951)
       at java.lang.reflect.Method.invoke(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:372)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1388)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1183)
```

**Comment by marwendoukh** [comment]
How to reproduce ?

**Comment by nbradbury** [comment]
Closed via #4519
